src/index.html

```diff
@@ -10,7 +10,58 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
```

Comment:
Fix formatting here:
```suggestion
    <form
      action="/index.html"
      method="post"
      data-qa="big"
      class="search-form"
     >
```